pkg/services/accesscontrol/roles.go

```diff
@@ -147,5 +198,23 @@ var PredefinedRoleGrants = map[string][]string{
 	RoleGrafanaAdmin: {
 		usersAdminEdit,
 		usersAdminRead,
+		orgsAdminEdit,
+		orgsAdminRead,
+		ldapAdminEdit,
+		ldapAdminRead,
 	},
+	string(models.ROLE_ADMIN): {
```

Comment:
Admins should be able to manage orgs they are admin for, see https://github.com/grafana/grafana/blob/master/pkg/api/api.go#L206